### ISSUE TITLE
Not able to create a localhost city for FOSS Hack 2025

### ISSUE THREAD

**Issue by fossdot** [fossunited/fossunited#724]
### Describe the bug

Following is the screenshot & copy-pasted error from the desk.

![Screenshot(201)](https://github.com/user-attachments/assets/72e7c3c9-75a4-4478-bc82-6f5dbd32db5b)

### App Versions
```
{
	"ballot": "0.0.1",
	"builder": "1.12.5",
	"fossunited": "0.0.1",
	"frappe": "15.50.0",
	"frappe_discourse": "0.0.1",
	"print_designer": "1.4.3",
	"wiki": "2.0.0"
}
```
### Route
```
Form/FOSS Hackathon LocalHost/new-foss-hackathon-localhost-julazxpisy
```
### Traceback
```
Traceback (most recent call last):
  File "apps/frappe/frappe/app.py", line 114, in application
    response = frappe.api.handle(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/api/__init__.py", line 49, in handle
    data = endpoint(**arguments)
           ^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/api/v1.py", line 36, in handle_rpc_call
    return frappe.handler.handle()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/handler.py", line 49, in handle
    data = execute_cmd(cmd)
           ^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/handler.py", line 85, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/__init__.py", line 1725, in call
    return fn(*args, **newargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/utils/typing_validations.py", line 31, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/desk/form/save.py", line 39, in savedocs
    doc.save()
  File "apps/frappe/frappe/model/document.py", line 342, in save
    return self._save(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 364, in _save
    return self.insert()
           ^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 295, in insert
    self.run_before_save_methods()
  File "apps/frappe/frappe/model/document.py", line 1104, in run_before_save_methods
    self.run_method("before_save")
  File "apps/frappe/frappe/model/document.py", line 974, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1334, in composer
    return composed(self, method, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 1316, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/model/document.py", line 971, in fn
    return method_object(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/fossunited/fossunited/foss_hackathon/doctype/foss_hackathon_localhost/foss_hackathon_localhost.py", line 45, in before_save
    len(self.organizers) > len(self.get_doc_before_save().organizers)
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'organizers'

```
### Request Data
```
{
	"type": "POST",
	"args": {
		"doc": "{\"docstatus\":0,\"doctype\":\"FOSS Hackathon LocalHost\",\"name\":\"new-foss-hackathon-localhost-yrolatmjqj\",\"__islocal\":1,\"__unsaved\":1,\"owner\":\"vishal@fossunited.org\",\"is_accepting_attendees\":0,\"organizers\":[],\"__run_link_triggers\":false,\"parent_hackathon\":\"o312an73dc\",\"localhost_name\":\"Delhi-NCR\",\"city\":\"Delhi\",\"state\":\"Delhi/NCR\",\"location\":\"TBD\",\"custom_venue_image\":\"\",\"custom_forum_link\":\"https://forum.fossunited.org/t/foss-hack-localhost-dtc-greater-noida/3258\",\"map_link\":\"https://maps.app.goo.gl/bGQPut52fHVHoLKU7\",\"creation\":\"\",\"modified_by\":\"vishal@fossunited.org\",\"modified\":\"\",\"lft\":null,\"rgt\":null,\"idx\":null}",
		"action": "Save"
	},
	"freeze": true,
	"headers": {},
	"error_handlers": {},
	"url": "/api/method/frappe.desk.form.save.savedocs",
	"request_id": "0ee813ee-7312-4544-93aa-530d3fc27d42"
}
```
### Response Data
```
{
	"exception": "AttributeError: 'NoneType' object has no attribute 'organizers'",
	"exc_type": "AttributeError",
	"_exc_source": "fossunited (app)"
}
```

**Comment by harshtandiya** [comment]
This is happening because a localhost is supposed to be created with atleast 1 organizer

**Comment by fossdot** [comment]
Still the same error.

![Screenshot(205)](https://github.com/user-attachments/assets/700b1d95-c4d8-4163-9c74-37005e98c114)
![Screenshot(206)](https://github.com/user-attachments/assets/4ba8bc51-d497-4824-b17d-cdb6821837c0)

**Comment by fossdot** [comment]
@harshtandiya any update on this?

**Comment by harshtandiya** [comment]
closed with #743 

site will be update today if Pending PRs are merged
